From df332551623629eb5373bdc7a8fa125a128c4351 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Corentin=20No=C3=ABl?= <corentin@elementary.io>
Date: Mon, 2 Mar 2020 12:15:57 +0100
Subject: [PATCH 21/48] cogl: Add main header for Cogl

This allows bindings linking to the C header to actually have the right one.

https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1101
---
 cogl/cogl/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cogl/cogl/meson.build b/cogl/cogl/meson.build
index 0297a2485..be93ba2f9 100644
--- a/cogl/cogl/meson.build
+++ b/cogl/cogl/meson.build
@@ -478,6 +478,7 @@ if have_introspection
       '-D__COGL_GLX_H_INSIDE__',
       '-DCOGL_GIR_SCANNING',
     ],
+    header: 'cogl/cogl.h',
     install_dir_gir: pkglibdir,
     install_dir_typelib: pkglibdir,
     install: true
-- 
2.26.0.rc2

